### ISSUE TITLE
AI units "give up"

### ISSUE THREAD

**Issue by greenfield** [OpenRA/OpenRA#10150]
I observed this in a mission ("06b: Bridge over the River VizchGoi") after having taken over the enemy base. The enemy tanks positioned on the bridge didn't start attacking even though I got really close with my units. Is this intended behavior?

![tame](https://cloud.githubusercontent.com/assets/1336770/11464935/336e6528-96e9-11e5-945e-6af3ac941eca.png)

**Comment by abcdefg30** [comment]
I think that could be a case of #7539 / #9378.
The tanks probably tried to attack another target, but couldn't move to it,
because of the bugs posted above.

In addition: The tanks aren't lua controlled.

**Comment by hypercube33** [comment]
No, this happens sometimes during a regular game, depending on the day. It seems a lot better lately where the AI keeps trying to play after a while of time in-game goes by. They never expand though, so we need that fixed at the least.

**Comment by obrakmann** [comment]
@greenfield: can you please provide the replay?

@hypercube33: please don't mistake the mission AI for the skirmish AI. They're completely separate things.

**Comment by greenfield** [comment]
@obrakmann Sorry for the delay. I found the replay. It's rather embarrassing, but if it helps to figure out what was going on, here it is:

[OpenRA-2015-11-30T053623Z.orarep.zip](https://github.com/OpenRA/OpenRA/files/53088/OpenRA-2015-11-30T053623Z.orarep.zip)
(The weird enemy behavior is visible beginning at 20mins40sec.)
